### PR TITLE
Add my portion of About Us

### PR DESCRIPTION
**Pull request by leezhixuan**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -21,13 +21,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Jane Doe
```
Comment (Justinhoejj) [inline docs/AboutUs.md:22]: Should this be replaced with your name?

**Review by Justinhoejj — APPROVED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (Justinhoejj) [reply docs/AboutUs.md:22]: Otherwise looks good to merge

File: docs/team/leezhixuan.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Lee Zhi Xuan's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (Jiwei99) [inline docs/team/leezhixuan.md:6]: Might want to update to fit our app i.e. CONNECTIONS

**Review by Jiwei99 — APPROVED** [review]
Lgtm

**Pull request merged**